db/tgif.sql

```diff
@@ -25,7 +25,7 @@ CREATE TABLE proposals
     amount_requested FLOAT,
     link VARCHAR,
     description_text VARCHAR, 
```

Comment:
syntax error: need to remove the comma here for the sql file to compile properly!